css-view-transitions-2/Overview.bs

```diff
@@ -434,32 +434,44 @@ The {{CSSViewTransitionRule}} represents a ''@view-transition'' rule.
 ### Setting up the view-transition in the old {{Document}} ###  {#setup-old-document-vt}
 
 	<div algorithm>
-		To <dfn export>setup cross-document view-transition</dfn> given a {{Document}} |oldDocument|,
-		a {{Document}} |newDocument|, a {{NavigationType}} |navigationType|, a boolean |isBrowserUINavigation|, and |onReady|, which is an algorithm accepting nothing:
+		To check if a <dfn export>should navigation trigger a cross-document view-transition?</dfn> given
```

Comment:
Also a note would be nice to clarify that this algorithm is to check whether a navigation qualifies for a ViewTransition assuming the Document has opted-in.

Reply:
Done